service/application/src/main/java/com/solace/maas/ep/runtime/agent/config/plugin/enumeration/MessagingServiceType.java

```diff
@@ -7,5 +7,8 @@ public enum MessagingServiceType {
     SOLACE,
 
     @JsonProperty("Kafka")
-    KAFKA
+    KAFKA,
```

Comment:
I assume we want to get rid of this soon? Enum and plugins don't mix well.

Reply:
Yes actually I removed the entire Enum it's not being used anywhere except for the Event produced by EP. I didn't remove it from there because I didn't want to break the EP messaging implementation. I'll let the Developers decide when they want to remove it.